flow/core/experiment.py

```diff
@@ -153,6 +166,14 @@ def rl_actions(*_):
                 vel.append(np.mean(self.env.k.vehicle.get_speed(veh_ids)))
                 ret += reward
 
+                # collect additional information for the data pipeline
+                get_extra_info(self.env.k.vehicle, extra_info, veh_ids, source_id, run_id)
+
+                # write to disk every 100 steps
+                if convert_to_csv and self.env.simulator == "traci" and j % 100 == 0:
+                    write_dict_to_csv(trajectory_table_path, extra_info, not j)
```

Comment:
I'm sorry, can you change this line to `write_dict_to_csv(trajectory_table_path, extra_info, not j and not i)`, please. This is my bad, I forgot to take into consideration of multi-runs. The current code will write a header to the CSV for each run.